src/bundle/Resources/public/js/scripts/admin.notifications.list.js

```diff
@@ -0,0 +1,174 @@
+(function (global, doc, ibexa, Translator, Routing) {
```

Comment:
What do you think about using import for `Translator`, `Routing`, `showErrorNotification`, `getJsonFromResponse` and also for `getRequestMode`, `getRequestHeaders`
```suggestion
import { getTranslator, getRouting } from '@ibexa-admin-ui/src/bundle/Resources/public/js/scripts/helpers/context.helper';
import { getRequestHeaders, getRequestMode, getJsonFromResponse } from '@ibexa-admin-ui/src/bundle/Resources/public/js/scripts/helpers/request.helper.js';
import { showErrorNotification } from '@ibexa-admin-ui/src/bundle/Resources/public/js/scripts/helpers/notification.helper';

(function (global, doc, ibexa) {
```

Reply:
After internal discussion, we decided not to use imports, since this module is only for BO and changes will go to 4.6.